systemtest/otherLoadTest/playlist.xml

```diff
@@ -21,6 +22,61 @@
 		<groups>
 			<group>system</group>
 		</groups>
+		<impls>
+			<impl>openj9</impl>
+		</impls>
+	</test>
+	<test>
+		<testCaseName>ClassLoadingTest_HotSpot_Non_zLinux</testCaseName>
```

Comment:
Somewhere there should be comments explaining why the need to temporarily separate into 2 targets, and a link to the defect/issue that caused this change.

Reply:
I think it's a bit hard to see this in the change view. 

We have 3 Mauve load tests - single thread, single invocation and multi-thread. Since we need the exclude for each one of them, the split has taken places in all 3 of them. So we ended up with three blocks of mauve tests, where in each block we have 3 `<test>` nodes. 

There is a comment with the link to the issue at the beginning of each such block.

Reply:
Yes, I see now when I view the full file, thanks...  

this PR is tricky to review when just looking at the default review mode.  :)

Reply:
Yes... I understand .. since there are 3 pairs of tests whose names are very similar ... sorry about that..

Reply:
no worries, I know its temporary :)